buildutils/src/ensure-repo.ts

```diff
@@ -250,6 +249,42 @@ export async function ensureIntegrity(): Promise<boolean> {
     locals[data.name] = pkgPath;
   });
 
+  // Build up an ordered list of CSS imports for each local package.
+  Object.keys(locals).forEach(name => {
+    const data = pkgData[name];
+    const deps: Dict<string> = data.dependencies || {};
+    const skip = SKIP_CSS[name] || [];
+    const cssData: Dict<Array<string>> = {};
+
+    if (data.jupyter && data.jupyter.extraStyles) {
```

Comment:
Should this not be `data.jupyterlab`? I see there are some extra styles missing somewhere (codemirror / xterm), while others are included (blueprint).

Reply:
I think you're right.

Reply:
Fixed in #6541